src/api/sizeme-api.js

```diff
@@ -408,6 +408,22 @@ function match (doSelectBestFit = true) {
                             .map(([sku, res]) => ({ [skuMap.get(sku)]: res }))
                     );
                 }
+                // time for the single fitted pantWaist exception
+                // run thru all results and see if we have a single pantWaist entered
+                const optFit = product.item.fitRecommendation ? product.item.fitRecommendation : DEFAULT_OPTIMAL_FIT;
+                Object.entries(result).forEach(([key, singleResult]) => {
+                    if (singleResult.matchMap) {
+                        if ((Object.keys(singleResult.matchMap).length === 1) &&
```

Comment:
Maybe prettier:
`if (singleResult.matchMap.pant_waist && Object.keys(singleResult.matchMap).length === 1`

Also, you could combine that line with the surrounding if-clause to reduce the levels

Reply:
Yeah, I think there was also one line too long.